website/static/js/privateLinkManager.js

```diff
@@ -13,11 +13,32 @@ var PrivateLinkViewModel = function(url) {
     self.anonymous = ko.observable(false);
     self.pageTitle = 'Generate New Link to Share Project';
     self.errorMsg = ko.observable('');
+    self.id = ko.observable('');
 
     self.nodes = ko.observableArray([]);
     self.nodesToChange = ko.observableArray();
     self.disableSubmit = ko.observable(false);
     self.submitText = ko.observable('Submit');
+
+    self.isChildVisible = function(data) {
```

Comment:
Take a look at website/templates/project/modal_generate_private_link.mako:53 (and one other line a few farther down)